hack/chains/deploy_local_rekor.tpl

```diff
@@ -0,0 +1,31 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: rekor
+spec:
+  destination:
+    # Define name or server, not both
+    name: ''
+    namespace: rekor-server
+    server: 'https://kubernetes.default.svc'
+  source:
+    path: helm-charts/rekor
+    repoURL: 'https://github.com/sigstore/sigstore-helm-operator'
+    targetRevision: HEAD
```

Comment:
Should we pin to a particular version for better reproducibility? I guess the downside is that we won't pick up updates automatically which might be desirable in a dev environment.